CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Added `ascending` parameter to `fetch_*_table()` methods ([#1602](https://github.com/neptune-ai/neptune-client/pull/1602))
 - Added `progress_bar` parameter to `fetch_*_table()` methods ([#1599](https://github.com/neptune-ai/neptune-client/pull/1599))
 - Added `progress_bar` parameter to `download()` method of the `Handler` class ([#1620](https://github.com/neptune-ai/neptune-client/pull/1620))
+- Added `progrees_bar` parameter to `fetch_values` method of the `Handler` class ([#1633](https://github.com/neptune-ai/neptune-client/pull/1633))
```

Comment:
```suggestion
- Added `progress_bar` parameter to `fetch_values()` method of the `Handler` class ([#1633](https://github.com/neptune-ai/neptune-client/pull/1633))
```

Reply:
Ah, good catch, it was late when I was writing it :D thank you!